util/ubaboot.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python3
+
```

Comment:
Rather than pick up the various whitespace formatting changes, its probably better to revert those and just ignore
```suggestion
# yapf: disable
```

Reply:
Done, but... what do you mean with "revert those whitespace formatting changes"? Are there changes in the use of spaces/tabs?

Reply:
Diff the contents with upstream, its fairly obvious.

Reply:
This shouldnt be marked as resolved when the requested actions have not been completed.

Reply:
Oh, ok... NOW I understand what you mean. I'll put the original file from the ubaboot repo (with the "retry" patch, of course).

Reply:
Ok, I think that now it's fixed.

Reply:
You still want the suggestion [here](https://github.com/qmk/qmk_firmware/pull/20351#discussion_r1161344778), so that CI ignores the file.

Reply:
Agh! I probably overwrote the change accidentally! Sorry!